kernel/Makefile

```diff
@@ -98,15 +106,17 @@ build_$(2)$(3): Dockerfile Makefile $(wildcard patches-$(2)/*) $(wildcard config
 		docker build \
 			--build-arg KERNEL_VERSION=$(1) \
 			--build-arg KERNEL_SERIES=$(2) \
-			--build-arg EXTRA=$(3) \
+			--build-arg EXTRA=`echo $(3) | sed -e 's/-dbg//'` \
```

Comment:
what is the reason for the `sed` gymnastics here? Wouldn't it be either to just another argument to the template?

Reply:
$(3) is involved in many places in Makefile where it's used to do tag, build command, or something else.  If we introduce $(4) with '-rt', that means we have to modify a lot of places.

Reply:
I don't mind it being modified in a lot of places in the makefile. I find it inconsistent to have have them as separate variables  in the Dockerfile but as one string in the Makefile which then is split up...